src/test/datascience/notebook/helper.ts

```diff
@@ -206,9 +206,6 @@ export async function closeNotebooksAndCleanUpAfterTests(disposables: IDisposabl
         // Dispose any cached python settings (used only in test env).
         configSettings.JupyterSettings.dispose();
     }
-    if (!isInsiders()) {
```

Comment:
Tests run on both stable & insiders.
All notebook tests wouldn't have been running even in stable because of this.
Removed this condition